account/identity.go

```diff
@@ -63,6 +106,8 @@ type IdentityRepository interface {
 	Save(ctx context.Context, identity *Identity) error
 	Delete(ctx context.Context, id uuid.UUID) error
 	Query(funcs ...func(*gorm.DB) *gorm.DB) ([]*Identity, error)
+	List(ctx context.Context) (*app.IdentityArray, error)
+	ValidIdentity(context.Context, uuid.UUID) bool
```

Comment:
@alexeykazakov IMHO, I'd name it `IsValid(context.Context, uuid.UUID) bool` cause it is known that you're handling Identities. That is similar to what we have with the rest of methods `Delete(, List(`.